api/prisma/seed.ts

```diff
@@ -145,32 +143,29 @@ export const initialProdSeed = async (): Promise<void> => {
                     cleanContent: htmlToText.convert(problem.content)
                 }
             });
-        }    
-
+        }
     }
-
 };
 
 switch (process.env.STAGE) {
```

Comment:
This switch should be wrapped in a function

Reply:
Actually, ignore this atm because this file needs to be refactored anyways.
There's a good example on prisma docs on how this file should look like: https://www.prisma.io/docs/guides/database/seed-database

Reply:
This file wasn't changed I think except @Harry-Jisc  ran prettier on it